### ISSUE TITLE
Id variable ruins the model 

### ISSUE THREAD

**Issue by GioLomia** [GioLomia/Deep_Learner#5]
If the id variable is present in the data set, it will ruing the predictive model. Need some kind of filter algorithm to find the ID variable and take it out, no matter what it is called or where it is.

**Comment by Radu1997** [comment]
line 214 needs to be
df = pd.DataFrame.to_csv(path,index=False)
(i commented on your post in r/deeplearning)

**Comment by GioLomia** [comment]
True. But I was more specifically talking about internal ID variable. Almost all kaggle data sets contain id column on their own. I was thinking we could automatically get rid of it somehow.

**Comment by Radu1997** [comment]
Right, I get what you mean. Indeed a bit tricky to provide a consistent solution. 
I was thinking, something along the lines of a NN that measures graph isomorphism, we could use it to compare the first column of the dataset with the rest of the matrix. If the correlation is close to 0, we can deduce that we're talking about an index-column.
https://www.google.com/amp/s/www.researchgate.net/publication/251044780_The_Decision-Making_Neural_Networks_Model_for_Solving_the_Graph_Isomorphism_Problem_The_Decision-Making_Neural_Networks_Model_for_Solving_the_Graph_Isomorphism_Problem/amp

**Comment by Radu1997** [comment]
I get what you mean. Creating an enum-object with the same length as the amount of tuples as the data matrix and comparing its values to the the first columns. But that might not be consistent.

**Comment by GioLomia** [comment]
I was also thinking we could try parse through column titles and fin if it has *id* in it somewhere, but then it will also spot variable names like industry_id and stuff like that so it is a problem.
